src/Misc.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  Typography,
+  Card,
+  Paper,
+  FormControl,
+  RadioGroup,
+  FormControlLabel,
+  Radio,
+  TextField,
+  MenuItem,
+  Checkbox,
+  InputBase,
+  Chip,
+  Autocomplete,
+  Avatar,
+  Dialog,
+  InputLabel,
+  Select,
+  Alert,
+  Button,
+  IconButton as MuiIconbutton,
+} from "@mui/material";
+
+export {
+  RadioGroup,
+  Typography,
+  Card,
+  MenuItem,
+  FormControl,
+  FormControlLabel,
+  Radio,
+  Checkbox,
+  InputBase,
+  TextField,
+  Paper,
+  Chip,
+  Autocomplete,
+  Avatar,
+  Dialog,
+  InputLabel,
+  Select,
+  Alert,
+  Button,
+  MuiIconbutton,
+};
```

Comment:
I think these can just go straight into index.ts and use

`export {} from "@mul/material"` syntax